app/views/vocabs/index.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.array! @articles do |article|
+  json.id article.id
```

Comment:
modify `@articles` for `@vocabs` or whichever variable you send from the controller